components/style/invalidation/element/invalidator.rs

```diff
@@ -384,23 +377,23 @@ where
                 sibling_invalidations,
             );
 
+        let invalidated_descendants = child_invalidator.invalidate_descendants(
+            &invalidations_for_descendants
+        );
+
         // The child may not be a flattened tree child of the current element,
         // but may be arbitrarily deep.
         //
         // Since we keep the traversal flags in terms of the flattened tree,
         // we need to propagate it as appropriate.
```

Comment:
This comment seems to be describing what we do inside the `invalidated_descendants` implementation, which could be anything on some abritrary InvalidationProcessor.  This comment is already duplicated in the (one) `invalidated_descendants` implementation so I think we should just drop this comment here.